plugin/pkg/scheduler/algorithm/BUILD

```diff
@@ -24,6 +24,7 @@ go_library(
         "//plugin/pkg/scheduler/schedulercache:go_default_library",
         "//vendor:k8s.io/apimachinery/pkg/apis/meta/v1",
         "//vendor:k8s.io/apimachinery/pkg/labels",
+        "//vendor:k8s.io/client-go/pkg/apis/apps",
```

Comment:
@bsalamat ,  seems you forgot my previous question for you PR: Why not use `apps` as `k8s.io/kubernetes/pkg/apis/apps/v1beta1` here? I saw that many other places are using `apps` as `k8s.io/kubernetes/pkg/apis/apps/v1beta1` for StatefulSet.